system-test/bigtable.js

```diff
@@ -1035,17 +1035,48 @@ describe('Bigtable', function() {
 
         row.delete(done);
       });
+    });
 
-      it('should delete all the rows', function(done) {
-        TABLE.deleteRows(function(err) {
-          assert.ifError(err);
+    describe('.truncate()', function() {
```

Comment:
Did you see the change in #101?

Reply:
Yup, decorating the updated test with `skip`